Gopkg.lock

```diff
@@ -2,14 +2,175 @@
 
 
 [[projects]]
+  branch = "master"
+  name = "github.com/Azure/go-ansiterm"
+  packages = [".","winterm"]
+  revision = "fa152c58bc15761d0200cb75fe958b89a9d4888e"
+
+[[projects]]
+  name = "github.com/Microsoft/go-winio"
```

Comment:
FWIW this is ultimately dragged in by using `alltransports.ParseImageName`, which roughly amounts to “I want to be able to consume any kind of image, let the user specify a `"`_transport_`:`_something_`"` string which I don’t understand and just pass along to the library and let the library do everything that is needed”.

I do think that such an ability to inspect an image wherever it is stored can be very useful, but if you were definitely restricted to docker/distribution-hosted images by some other factor, replace `alltransports.ParseImageName` with `containers/image/docker.ParseReference` (using the after-`:` syntax, roughly `"//example.com/ns/busybox:latest"`); that would allow you to get rid of quite a few dependencies.